(DOCS-2753) Remove instances of 'we' from docs (#12609)
DataDog/documentation: content/en/tracing/guide/week_over_week_p50_comparison.md

    This page contains a list of all instrumented [services][2] available in Datadog APM. You can search over your services by keywords, filter them by `env` tag, and set the timeline.
2. **Search and open a relevant and active service**.

>     The web-store service is used in this example because it’s a stable service that should be double checked that there hasn’t been any issues over the last month.

Reviewer: ```suggestion
    This example uses the web-store service because it is stable. Double-check that issues have not appeared over the last month.
```
